### DISCUSSION TITLE
Warning messages covering page

### DISCUSSION THREAD

**Discussion by Serendibb** [burkybang/SuiteAdvanced-Feedback#185] [General]
When I'm editing A.PDFs, the Messages keeps staying on screen for me. Just want to check if this is happening to anyone else please? 
![image](https://github.com/user-attachments/assets/dba17a62-4004-4e5e-ace8-b713a475dec0)

**Comment by burkybang** [comment]
This is due to a bug in NetSuite's source code that was introduced a couple years ago. You can enable the banners to auto dismiss in SuiteAdvanced settings, or you can manually close them by hovering and clicking the X.

**Reply by Serendibb** [reply]
Arh, great! I found the settings to dismissed it! Just too many functions. Do consider defaulting it to like 5 or 10 seconds in the future release please.

**Reply by burkybang** [reply]
My thinking is this: If I have an option that changes how NetSuite behaves, I'd rather allow the user to opt in to the change rather than assume they'd like it. This only applies to features that can be enabled or disabled of course.
